operator/src/test/java/oracle/kubernetes/operator/wlsconfig/WlsClusterConfigTest.java

```diff
@@ -241,17 +248,16 @@ public void verifyMaxClusterSizeForDynamicCluster() {
   @Test
   public void verifyMaxClusterSizeForMixedCluster() {
     WlsDynamicServersConfig wlsDynamicServersConfig =
-        createDynamicServersConfig(1, 1, "ms-", "cluster1");
+        createDynamicServersConfig(1, 1, 1,"ms-", "cluster1");
     WlsClusterConfig wlsClusterConfig = new WlsClusterConfig("cluster1", wlsDynamicServersConfig);
     wlsClusterConfig.addServerConfig(createWlsServerConfig("ms-0", 8011, null));
 
     assertThat(wlsClusterConfig.getMaxClusterSize(), equalTo(2));
   }
 
-  @Test
   public void verifyGetUpdateDynamicClusterSizeUrlIncludesClusterName() {
     WlsDynamicServersConfig wlsDynamicServersConfig =
-        createDynamicServersConfig(1, 1, "ms-", "cluster1");
+        createDynamicServersConfig(1, 1, 1,"ms-", "cluster1");
```

Comment:
space after comma - weird that checkstyle did not get that

Reply:
I'll check... I wonder if Checkstyle is somehow not running again for test code.

Reply:
added space after comma after minClusterSize parameter for all createDynamicServersConfig() methods.